Components/avatar/avatar.html

```diff
@@ -0,0 +1,31 @@
+<!-- Avatars default - different Sizes -->
+
+<div class="avatar-xl-box">
+    <img src="avatar.jfif" alt="" class="avatar-xl">
+</div>
+
+<div class="avatar-l-box">
+    <img src="avatar.jfif" alt="" class="avatar-l">
+</div>
+
+<div class="avatar-s-box">
+    <img src="avatar.jfif" alt="" class="avatar-s">
+</div>
+
+<div class="avatar-xs-box">
+    <img src="avatar.jfif" alt="" class="avatar-xs">
+</div>
+
+<!-- Avatars with Status -->
+
+<div class="avatar-xl-box online">
+    <img src="avatar.jfif" alt="" class="avatar-xl">
+</div>
+
+<div class="avatar-l-box offline">
+    <img src="avatar.jfif" alt="" class="avatar-l ">
+</div>
+
+<div class="avatar-s-box sleep">
+    <img src="avatar.jfif" alt="" class="avatar-s">

```

Comment:
Suggestion: Please provide alt value for accessibility checkpoints.

Reply:
Actually I wanted to leave that to the users, since they could be using the component for various purposes.

Reply:
Sure, they will change it accordingly to their use case. However, you should put a value here say- " dummy avatar" for screen readers to understand this as well. 
Another scenario is when your image couldn't load for whatever reasons, it will display the alt text. Without which it would be hard to figure out that there is an image also.